### PR TITLE
RPED cost reduction

### PR DESCRIPTION
**Pull request by iFlashYou**
This PR adjusts the price of the RPED to reduce the iron and glass cost and to add gold to the price. The primary reasoning behind this is to reduce the construction time of the RPED (nearly 2 minutes) and since about nearly 70 sheets of metal and glass for a simple tool that makes upgrading possible without taking 40 minutes (try to upgrade all of medbay using a gadget bag, screwdriver, and crowbar and see how long it takes for you to get chloraled because a doctor thinks your sabotaging their equipment). I added the gold so you have to wait for the miners to deliver materials first before you can build it. Like I said before, its mostly due the long time it takes to build it , I understand it for weapons and other stuff where you can harm people or the station but you can not do anything harmfull with the repd so I dont understand this high cost for it.

🆑 
* tweak: Lowered RPED material price in the protolathe and added gold to its new price. New price: 7000 Iron, 3500 Glass and 600 Gold

### CODE REVIEW

**Comment by yclatious** [comment]
To give you some history, local github idiot jakmak decided that just science getting the toy mechanics should use was bad since no miners, unless bitched at, deliver to mechanics, as such, he removed the miner mineral requirement but lo and behold, coders so declared it power crepe and made it take whole minutes to make, to nobody's real surprise or outrage.

And now, we go back to square one, as we again, get the mineral requirement, decent non stupid time and with people now sad they have to use gadget bags when this tool should have never really had any sort of miner ore requirement or massive wait time, just a science requirement of maybe bluespace and programing and a slightly steeper prime buff like maybe a RPED but what the fuck do we know, back to minerals.

**Comment by Kurfursten** [comment]
Previous discussion #14481

And the original cost of the RPED, before gold and silver were removed in exchange for higher metal/glass can be seen here: https://github.com/vgstation-coders/vgstation13/pull/12660/files

The main effect of having this cost gold only instead of just silver is that you could rob the vault to make an RPED before minerals come in.

**Comment by SonixApache** [comment]
[needs changelog]
make the commit message not shit

**Comment by N3X15** [comment]
Added [Balance] because this affects availability of an item.

**Comment by DrSnips** [comment]
I dont see why it should cost anything special, its a tool to do a job, not even the actual resources to do the job, stock parts. The material useage should be there and not the rped. Hypothetically giving them one or three even at round start wouldn't change anything, they need the research and materials for t3 parts (no one upgrades with t2 parts, come on now.) I could supply medical completely with the adv tools and kits for under the cost of the rped and parts and they would get more out of them.
The amount of gold isnt bad, but still seems like an excuse for nerfing the cost a good amount. I'd just make it the planned metal and glass and call it a day.

**Comment by PJB3005** [comment]
> its a tool to do a job

Just like a mining drill!

The whole point of the RPED price is to gate it to take longer to get into a round. In the mean time science can upgrade some stuff like their lathe manually to reduce costs.

**Comment by ghost** [comment]
Same issue as the last ten billion times the RPED costing rares came up. The literal abortions infrequently get minerals and unlike actual rnd scientists, they have to interaction meme and hope a HoP/Captain/miner gives a single solitary fuck about them instead of just going and mining themselves. 2 minutes of build time and a little looting or deconstruction of resources that are everywhere is head and shoulders above not getting it at all. The workaround is a coin flip to actually get and it's gonna get unfeature'd inevitably when they realize it makes the whole thing pointless so don't start.

@yclatious The solution has always been to add another sci alt-title, delete mechanics, put the flatpack, blueprint, gen fab, and reverse engine into rnd so there's none of the stupid needless friction and mineral fighting in the first place solely because people would break down in tears then suicide succumb if someone said thanks scientist after a machine was installed or upgraded. Not wanting to be associated with the purple tide when you did part of rnd work - upgrading and installing machinery - was never a solid reason for all the issues that came out of a duplicate job.

**Comment by SonixApache** [comment]
MECHANICS IS THE OPS DIVISION OF SCIENCE

**Comment by yclatious** [comment]
@SarahJohnson489 I will only ever accept that change if mechanics continue with their titles, keep all their machines like you said and if the machine restricts access to non mechanics, fuck giving the autism bunker any credit under any circumstance, id rather keep a broken job like this and give minerals exclusively to a duplicate job than enable the lazy asses of science.

All hail Mechania, fuck Sciencia.

**Comment by FudgePucker** [comment]
@SarahJohnson489
How about we remove R&D and have mechanics fill that role?
Science has so much shit and 95% of the science players would rather suicide succumb and throw their computer out a window instead of helping another player.
This would have two benefits.
First,  miners would only need to stop by one area, no more fighting for mats! It is also closer to cargo so miners don't have to go as far for their toys. Also who has more use for the over flow metal and glass? Engineering or science? Science players would rather nonantag 6 bomb and get banned for it before they would let a single meesly sheet of iron leave the department.
Second, no more guns for science outside of xenoarch.

**Comment by Kurfursten** [comment]
@FudgePucker I actually wouldn't mind that idea, if Spacepod Mechanic sort of became a separate thing or went away entirely since people who choose that title typically end up leaving the station rather than remaining around to work the protolathe.

**Comment by DrSnips** [comment]
Here's a question, if you as a mechanic are tired of not getting materials and just want to work the proto/do upgrades, why wouldn't you just switch to your job to scientist? It seems like an easy fix to a personal problem most mechanics tell. Or could there possibly be something else mechanics really want to do that isn't actually do work for others and those listed tasks are actually just excuses to keep the job around?

**Comment by ghost** [comment]
@FudgePucker No, it's still causing a mineral split issue with robotics. Like I said, move the shit into RnD, give them mechanic as an alt-title. At least people with false expectations will fuck off instead of throwing a fit that someone who only did the bare minimum of rnd they needed for something else wasn't doing the actual rnd job. Nobody has a conniption when the mechanic is a useless greytiding fucker or there's no mechanics at all but lord save you if you just did enough for a ddrill before you went large artifact hunting. Literally the second coming of satan purge the dept and monkey spike all of the purple men.

Miners are already stingy with metal/glass regardless which is why you buy it from cargo or get access regardless of if you're a scientist or a mechanic. And if you really want to get into the mineral meta, it's pointless to give engineering metal/glass anyway. They can order it like anyone else but unlike sci/mechanics who will actually use, read: waste, it, the only way you'll ever get to do a significant build is if you recall as many times as it takes and space cuffed/kill the people that try to stop it.

The xenoarch guns are by and large garbage. there's no sense spending half a round getting a ddrill and go prospecting for some inferior revolvers and half mag lasers

@DrSnips It's because they have the most horrific stubborn autism about being called a scientist. You'd think it was splashing acid on them when they hear it. It's a duplicate job for no good reason.

**Comment by SonixApache** [comment]
@FudgePucker 
Replacing R&D with the mechanics is some of the dumbest mechanics-related shit I've heard in a while.

The smart solution that doesn't revolve around underpriviledged not-engineers whining about scientists being scientists would be to just tack the mechanics lab to the R&D lab. Works out on most maps, solves a buncha problems.

**Comment by iFlashYou** [comment]
Just to remind you, this is about lowering the price of the RPED and not meachanics vs. scientist autism

**Comment by N3X15** [comment]
Changelog entry is wrong, RTFM.

**Comment by ghost** [comment]
The answer is still no for the stated reasons that a time gate that only punishes mechanics with an inconsistent and inevitably removed workaround while scientists can just go get the minerals themselves just fucks over the mechanics in all but the best of times. Loot or salvage station metal/glass or queue it up then go order metal and glass crates and it'll be done when you get back. 2 minutes and something you have an entire toolbox and some of is a lot more practical for the upgrade concerned user than rolling yahtzee on the access/miner/beating others to the workaround dice.

tl;dr, If the workaround trivializes the speedbump, don't even bother putting it in.

**Comment by jakmak6** [comment]
merge this shit it won a popular vote http://ss13.moe/index.php/poll/126

**Comment by Exxion** [comment]
Wow. That's a pretty decisive win.
